integrations/access/accessrequest/app.go

```diff
@@ -146,7 +149,7 @@ func (a *App) run(ctx context.Context) error {
 	}
 
 	if err := a.initAccessMonitoringRulesCache(ctx); err != nil {
-		return trace.Wrap(err)
+		log.WithError(err).Errorf("initialising Access Monitoring Rule cache")
```

Comment:
I'm not sure I understand this fix. Why are we ignoring any error from the initialization? We should only allow if it's a permission error watching access monitoring rules. Or check what resources are allowed like @marcoandredinis suggested below.